docs/pages/admin-guides/teleport-policy/database-access-controls.mdx

```diff
@@ -0,0 +1,120 @@
+---
+title: Teleport Policy and Database Access Controls
+description: Describes how to import and visualize advanced Teleport Database Access Controls policies using Teleport Policy and Access Graph.
+---
+
+Advanced Database Access Controls features allow Teleport administrators to configure user permissions for individual database objects using flexible, label-based approach for expressing access policies.
```

Comment:
Is the feature called Advanced Database Access Controls? If so, we should change this line to remove "features", since this is a single feature:

```markdown
Advanced Database Access Controls allow Teleport administrators to configure user permissions for individual database objects using flexible, label-based approach for expressing access policies.
```